flink-libraries/flink-table/src/main/scala/org/apache/flink/table/runtime/aggregate/ProcTimeBoundedRangeOver.scala

```diff
@@ -62,8 +62,8 @@ class ProcTimeBoundedRangeOver(
   private var function: GeneratedAggregations = _
 
   override def open(config: Configuration) {
-    LOG.debug(s"Compiling AggregateHelper: $genAggregations.name \n\n " +
-                s"Code:\n$genAggregations.code")
+    LOG.debug(s"Compiling AggregateHelper: ${genAggregations.name} \n\n " +
```

Comment:
I think we supported remove the " ", i.e.: 
`${genAggregations.name} \n\n "` -> `${genAggregations.name} \n\n"`
If we do this change, the log print info as follows:
FROM 
```
Compiling AggregateHelper: MinMaxAggregateHelper 

   Code:
GeneratedAggregationsFunction(MinMaxAggregateHelper,
```
TO
```
Compiling AggregateHelper: MinMaxAggregateHelper 

Code:
GeneratedAggregationsFunction(MinMaxAggregateHelper,
```